src/Platform/Microsoft.Testing.Platform/OutputDevice/Terminal/TerminalTestReporter.cs

```diff
@@ -562,6 +562,17 @@ private static void FormatErrorMessage(ITerminal terminal, FlatException[] excep
             return;
         }
 
+        if (outcome == TestOutcome.Skipped &&
```

Comment:
I don't feel like this is the right location for that. The skip message is not an error message IMO.